workspace/scan_runs.sh

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
I think we should either use the proper bash script with the for loop that we used during tb or delete this

Reply:
agreed. 

i will delete all tjmp2 specific scripts from the default workspace.